Python/Scrap_Latest_Torrents/README.md

```diff
@@ -0,0 +1,24 @@
+## Scrap Latest Torrents
+
+This script downloads the torrents in our Torrent client from the website https://rarbg.to/ from an entered label.
+
+### Requirements
+
+1.  You must have a torrent client, for example: uTorrent, BitTorrent, etc
```

Comment:
You can provide links to download uTorrent and BitTorrent here itself. Adding a hyperlink would be good IMO

Reply:
I add links